modules/swagger-codegen/src/main/java/io/swagger/codegen/languages/AbstractJavaCodegen.java

```diff
@@ -705,6 +706,20 @@ public void setParameterExampleValue(CodegenParameter p) {
     }
 
     @Override
+    public String toExampleValue(Property p) {
+        if(p.getExample() != null) {
+            return escapeText(p.getExample().toString());
+        } else {
+        	return super.toExampleValue(p);
+        }
+    }
+    
+    @Override
+    public Mustache.Compiler processCompiler(Mustache.Compiler compiler) {
+    	return compiler.escapeHTML(false);
```

Comment:
Instead of turning it off, what about using {{{ ... }}} instead to use the unescape value?

The change will have a lot of impacts to mustache templates used by Java AP clients as well as server stubs (spring. jaxrs, play, etc)